sbi/inference/base.py

```diff
@@ -279,7 +275,7 @@ def _converged(self, epoch: int, stop_after_epochs: int) -> bool:
         """
         converged = False
 
-        posterior_nn = self._posterior.net
+        posterior_nn = self._neural_net
```

Comment:
is this really the same? does the `self._neural_net` point to the same, potentially trained, `self._posterior.net`?

Reply:
As you write below, the `self._posterior` is only initialized after training.